### PR TITLE
Writer pass for "Remote debugging"

### PR DESCRIPTION
**Pull request by mikehoffms**
This PR doesn't merge into main, it merges into PR https://github.com/MicrosoftDocs/edge-developer/pull/2464

Rendered article for review:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/remote-debugging?branch=pr-en-us-2467 [[gh rendered](https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/wv2_remote_debugging_docs/microsoft-edge/webview2/how-to/remote-debugging.md)]

AB#43504774

### CODE REVIEW

File: microsoft-edge/webview2/how-to/remote-debugging.md [hunk 1]
```diff
@@ -1,103 +1,102 @@
...
+1.  Go to `edge://inspect`.  In the **Connect to a remote Windows device** text box, enter `http://<Device Portal URL>`, and then click **Connect to Device**.
 
-1. Verify you are able to successfully connect and your debuggable WebView2 is displayed.
+1.  Verify that you can successfully connect, so that your debuggable WebView2 control is displayed:
 
     ![Connect to WebView2](./media/connect-to-webview2.png)
 
-1. Click **inspect** and `Microsoft Edge DevTools` for the WebView2 will open.
+1.  Click **inspect**.  Microsoft Edge DevTools opens for the WebView2 control.
+
```
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/remote-debugging.md:92]: @maurawinstanley - add .png screenshot of DevTools

**Review by maurawinstanley — APPROVED** [review]

File: microsoft-edge/webview2/how-to/remote-debugging.md [hunk 2]
```diff
@@ -1,103 +1,110 @@
...
+1.  Go to `http://<Device Portal URL>/msedge`, such as `http://localhost:50080/msedge`, and make sure your WebView2 instance appears in the list:
 
-1. Launch your app.
+    ![Debuggable WebView2 Instance](./remote-debugging-images/debuggable-webviews.png)
 
-1. Navigate to `http://<Device Portal URL>/msedge` and verify your WebView2 instance appears in the list:
+1.  Go to `edge://inspect`.  In the **Connect to a remote Windows device** text box, enter `http://<Device Portal URL>`, such as `http://localhost:50080`, and then click **Connect to Device**.
 
-    ![Debuggable WebView2 Instance](./media/debuggable-webviews.png)
+1.  Verify that you can successfully connect, so that your debuggable WebView2 control, named **Edge**, is listed under your machine name:
```
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/remote-debugging.md:93]: @maurawinstanley - is the wv2 control usually named "Edge" in this UI?  Could there be multiple?

File: microsoft-edge/webview2/how-to/remote-debugging.md [hunk 3]
```diff
@@ -1,103 +1,110 @@
...
+    ![Enable Developer Mode](./remote-debugging-images/enable-developer-mode.png)
 
-    ![Enable Developer Mode](./media/enable-developer-mode.png)
+1.  Enable Device Portal: Open Windows **Settings**, select **Privacy & security** > For developers**, and then turn on the **Device Portal** toggle.
 
-1. Enable `Device Portal`:
+    If your operating system isn't up-to-date, you might get a message in the **Settings** window: "The Windows Developer Mode package couldn't be found in Windows Update, so remote deployment and Windows Device Portal are unavailable.  [Learn more](/windows/uwp/debug-test-perf/device-portal-desktop#failed-to-locate-the-package)."
 
-    Open **Windows Settings> Privacy & security > For developers**:
+1.  If you get such a message, update your system OS to the latest version, and then turn on the **Device Portal** toggle.
```
Comment (dan-wesley) [inline microsoft-edge/webview2/how-to/remote-debugging.md:37]: "system OS" spell out OS on first use: operating system (OS)

File: microsoft-edge/webview2/how-to/remote-debugging.md [hunk 4]
```diff
@@ -1,103 +1,110 @@
...
+    Or, if you aren't running the app locally, or if you need to connect over `HTTPS`, select **Configure a secure connection**, and then follow the steps in the page that you're redirected to.
 
-1. Verify the page you are redirected to is as follows:
+1.  Verify that the page you're redirected to is as follows:
 
-    ![Device Portal](./media/device-portal.png)
+    ![Device Portal](./remote-debugging-images/device-portal.png)
 
-1. Navigate to `http://<Device Portal URL>/msedge` and verify Remote Tools for Microsoft Edge are functioning. The page should load an empty list `[]`.
+1.  Go to `http://<Device Portal URL>/msedge` and verify Remote Tools for Microsoft Edge are functioning. The page should load an empty list: `[]`.
```
Comment (dan-wesley) [inline microsoft-edge/webview2/how-to/remote-debugging.md:61]: what's the remedial action if it doesn't load an empty list.

File: microsoft-edge/webview2/how-to/remote-debugging.md [hunk 5]
```diff
@@ -1,103 +1,110 @@
...
+To use Microsoft Edge DevTools to debug a desktop WebView2 WinUI 2 (UWP) app, use remote debugging.  Remote debugging is necessary because currently, the built-in DevTools in Microsoft Edge cannot be launched inside a store-signed WebView2 WinUI 2 (UWP) app.
 
-1. Install [Edge Canary](https://www.microsoftedgeinsider.com/en-us/download/canary).
+Attach `Microsoft Edge DevTools` remotely to a WebView2 WinUI 2 (UWP) app as follows:
 
-1. In `Edge Canary` navigate to `edge://inspect` and verify the page is supported in your version of the browser.
+1.  Install [Microsoft Edge Canary Channel](https://www.microsoftedgeinsider.com/download/canary).
 
-1. Enable `Developer Mode`:
+1.  In Microsoft Edge Canary, go to to `edge://inspect` and verify that the **Inspect with Developer Tools** page exists in your version of the browser:
```
Comment (dan-wesley) [inline microsoft-edge/webview2/how-to/remote-debugging.md:25]: just a general comment, which is probably a style thing based on our different doc sets. We only use the inline code backtick for inline code, not as a general highlighting tool.

File: microsoft-edge/webview2/how-to/remote-debugging.md [hunk 6]
```diff
@@ -1,103 +1,110 @@
...
+    
+    For example, if you're using the [WebView2 Sample UWP app](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/webview2_sample_uwp), you can set those environment variables by adding the following lines in the `Browser.xaml.cs` file:
     
-    `WEBVIEW2_RELEASE_CHANNEL_PREFERENCE = 1`
+    ```csharp
+    Environment.SetEnvironmentVariable("WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS", "--enable-features=msEdgeDevToolsWdpRemoteDebugging");
+    Environment.SetEnvironmentVariable("WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS", "WEBVIEW2_RELEASE_CHANNEL_PREFERENCE=1");
+    ```
+
+    Those are lines 33 and 34 in this screenshot:
```
Comment (dan-wesley) [inline microsoft-edge/webview2/how-to/remote-debugging.md:81]: what is "Those"?, code settings?

File: microsoft-edge/webview2/how-to/remote-debugging.md [hunk 7]
```diff
@@ -1,103 +1,110 @@
...
+Use `Remote Tools for Microsoft Edge` to enable [Microsoft Edge DevTools](/microsoft-edge/devtools-guide-chromium/landing/) debugging.
 
 
 <!-- ====================================================================== -->
-## Debugging Desktop WebView2 WinUI 2 (UWP) apps
+## Debugging desktop WebView2 WinUI 2 (UWP) apps
 
-These steps can be used to attach `Microsoft Edge DevTools` remotely to a WebView2 WinUI 2 (UWP) app.
-Remote debugging is necessary as a current limitation to WebView2 WinUI 2 (UWP) apps is the built-in `Microsoft Edge DevTools` cannot be launched inside a store signed WebView2 WinUI 2 (UWP) app.
+To use Microsoft Edge DevTools to debug a desktop WebView2 WinUI 2 (UWP) app, use remote debugging.  Remote debugging is necessary because currently, the built-in DevTools in Microsoft Edge cannot be launched inside a store-signed WebView2 WinUI 2 (UWP) app.
```
Comment (dan-wesley) [inline microsoft-edge/webview2/how-to/remote-debugging.md:19]: "currently" needed?
suggest contraction for cannot for consistency

File: microsoft-edge/webview2/how-to/remote-debugging.md (hunk 4 above)
Comment (maurawinstanley) [reply microsoft-edge/webview2/how-to/remote-debugging.md:61]: There is no remedial action, there will only be an empty list unless there is an application running that is debuggable by remote tools.

File: microsoft-edge/webview2/how-to/remote-debugging.md (hunk 6 above)
Comment (maurawinstanley) [reply microsoft-edge/webview2/how-to/remote-debugging.md:81]: Changed to "the".  This is referring to the environment variables.

File: microsoft-edge/webview2/how-to/remote-debugging.md (hunk 5 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/how-to/remote-debugging.md:25]: @dan-wesley fyi FTS on "edge://" in this repo shows such specialty URLs as using backticks

File: microsoft-edge/webview2/how-to/remote-debugging.md (hunk 4 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/how-to/remote-debugging.md:61]:
```
The page loads an empty list: `[]`, unless there's an application running that is debuggable by remote tools.
```

**Review by dan-wesley — APPROVED** [review]

**Pull request merged**